src/components/provider.js

```diff
@@ -23,6 +23,7 @@ const intlFormatPropNames = Object.keys(intlFormatPropTypes);
 const defaultProps = {
     formats : {},
     messages: {},
+    textComponent: 'span',
```

Comment:
Yeah, although I think given our earlier discussion on the propagation of css properties, it's good to be careful with automatic detection.